src/components/ModalWithForm.js

```diff
@@ -1,14 +1,15 @@
-import Modal from "./Modal.js";
+import Modal from "./Modal";
 
-export default class ModalWithForm extends Modal {
-  constructor(modalSelector, handleFormSubmit) {
-    super(modalSelector);
+class ModalWithForm extends Modal {
+  constructor({ modalSelector, handleFormSubmit }) {
+    super({ modalSelector });
     this._modalForm = this._modalElement.querySelector(".modal__form");
     this._handleFormSubmit = handleFormSubmit;
   }
 
   _getInputValues() {
     this._inputList = this._modalForm.querySelectorAll(".modal__form-input");
```

Comment:
**Could be improved**

All elements found **every time** you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `getView`) only **1 time** in order not to waste resources on searching them again and again.

`Class fields` (the so-called **class variables** with `this`) are accessible inside any class method. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.


Please, find it in line 7 only once